extensions/observability-devservices/deployment/src/main/java/io/quarkus/observability/deployment/ObservabilityDevServiceProcessor.java

```diff
@@ -76,6 +83,14 @@ public void startContainers(LaunchModeBuildItem launchMode,
             GlobalDevServicesConfig devServicesConfig,
             BuildProducer<DevServicesResultBuildItem> services) {
 
+        // To keep backwards compatibility we want to allow "http://${quarkus.otel-collector.url}" expression as a value
+        if (ConfigUtils.isPropertyPresent(OTLP_TRACES_ENDPOINT) &&
+                !ConfigProvider.getConfig().unwrap(SmallRyeConfig.class).getConfigValue(OTLP_TRACES_ENDPOINT).getRawValue()
+                        .equals(OTEL_COLLECTOR_URL_EXPRESSION)) {
+            log.infof("Observability dev services won't start due to property [%s] being defined", OTLP_TRACES_ENDPOINT);
```

Comment:
From my previous general comment (about `ObservabilityDevServiceProcessor` usage),
this looks like it could go directly into LGTM DevResource start()?
And it would just not start LGTM, where others - when we have them / if users have them - can still run, etc